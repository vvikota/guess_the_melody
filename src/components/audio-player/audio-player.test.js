import React from 'react';
import renderer from 'react-test-renderer';
import AudioPlayer from './audio-player.jsx';

const mock = {
  isPlaying: true,
  src: `./music/Jah_Khalib_angela.mp3`
};

it(`Audioplayer is correctly render`, () => {
  const tree = renderer
  .create(<AudioPlayer
    isPlaying={mock.isPlaying}
    onPlayButtonClick={jest.fn()}
    src={mock.src}
  />, {
    createNodeMock: () => {
      return {};
    }
  }).toJSON();

  expect(tree).toMatchSnapshot();
});

