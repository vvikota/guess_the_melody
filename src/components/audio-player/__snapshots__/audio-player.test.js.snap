// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Audioplayer is correctly render 1`] = `
Array [
  <button
    className="track__button track__button--play"
    disabled={true}
    onClick={[MockFunction]}
    type="button"
  />,
  <div
    className="track__status"
  />,
]
`;
